docs/METRICS.md

```diff
@@ -9,7 +9,8 @@ to create the correct structure:
 
 Each entry in the JSON file is formed by a SAP HANA SQL query (key) and the metrics/additional information (value). The additional information is composed by:
 
-* `enabled (boolean, optional)`: If the query is executed or not (`true` by default is the `enabled` entry is not set). If set to `false` the metrics for this query won't be exported.
+* `enabled (boolean, optional)`: If the query is executed or not (`true` by default is the `enabled` entry is not set). If set to `false` the metrics for this query won't be executed.
+* `hana_version_range (list, optional)`: The SAP HANA database versions range where the query is available (`[1.0.0]` by default). If the current database version is not inside the provided range, the query won't be executed. If the list has only one element, all versions beyond this value (this included) will be queried.
```

Comment:
we could  maybe provide an example with the a more explicit with a  a concrete example of the range ? in sake of clearness

Reply:
There is an example in the json example in the same file, but yes, it might be improved